src/sbvr-api/sbvr-utils.ts

```diff
@@ -394,7 +442,7 @@ export const validateModel = async (
 			const values = await getAndCheckBindValues(
 				{
 					vocabulary: modelName,
-					odataBinds: [],
+					odataBinds: bindsForAffectedIds(rule.bindings, request) as any,
```

Comment:
If there needs to be a cast I'd rather it be in the `bindsForAffectedIds` function directly